src/test/java/SQLServerDockerImageTest.java

```diff
@@ -4,7 +4,11 @@
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
 
-import java.sql.*;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
```

Comment:
Don't include this change with this PR. I'll be refactoring it, will take care of it then.

Reply:
Same as above

Reply:
Let's not do that. It's more work when we pollute commit history of files with no significance to the current change, especially when we trace code changes. Let's not set that as a practice.

Reply:
same as above